src/components/screens/Consul/Proposals/NewProposal.js

```diff
@@ -0,0 +1,361 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { Alert, TouchableOpacity, StyleSheet, ScrollView } from 'react-native';
+import { useForm } from 'react-hook-form';
+import { useMutation } from 'react-apollo';
+
+import { Input } from '../../../Consul';
+import { Wrapper, WrapperHorizontal, WrapperWithOrientation } from '../../../Wrapper';
+import { texts, colors } from '../../../../config';
+import { Button } from '../../../Button';
+import { Checkbox } from '../../../Checkbox';
+import { START_PROPOSAL, UPDATE_PROPOSAL } from '../../../../queries/Consul';
+import { ConsulClient } from '../../../../ConsulClient';
+import { LoadingSpinner } from '../../../LoadingSpinner';
+import { ScreenName } from '../../../../types';
+import { QUERY_TYPES } from '../../../../queries';
+import { RegularText } from '../../../Text';
+import { Label } from '../../../Label';
+import { SafeAreaViewFlex } from '../../../SafeAreaViewFlex';
+
+const text = texts.consul.startNew;
+const queryTypes = QUERY_TYPES.CONSUL;
+
+const kategorien = [
+  { name: 'Associations', id: 0, selected: false },
+  { name: 'Culture', id: 1, selected: false },
+  { name: 'Economy', id: 2, selected: false },
+  { name: 'Employment', id: 3, selected: false },
+  { name: 'Environment', id: 4, selected: false },
+  { name: 'Equity', id: 5, selected: false },
+  { name: 'Health', id: 6, selected: false },
+  { name: 'Media', id: 7, selected: false },
+  { name: 'Mobility', id: 8, selected: false },
+  { name: 'Participation', id: 9, selected: false },
+  { name: 'Security and Emergencies', id: 10, selected: false },
+  { name: 'Social Rights', id: 11, selected: false },
+  { name: 'Sports', id: 12, selected: false },
+  { name: 'Sustainability', id: 13, selected: false },
+  { name: 'Transparency', id: 14, selected: false }
+];
```

Comment:
this should be some english name for the variable and maybe its a constant in uppercase. will this object always be correct or is this some testing dummy data?

Reply:
the name has been updated to TagCategories. this data is spoken in a new proposal creation page to give users ideas for tags.

Reply:
as a constant following best practice guidelines i would suggest `TAG_CATEGORIES`, that was what i meant with uppercase